### PR TITLE
Rename dropStream to drop as we are not dropping a stream here

### PR DESCRIPTION
**Pull request by codeliner**

### CODE REVIEW

**Comment by codeliner** [comment]
ping @prolic

File: tests/Schema/SnapshotStoreSchemaTest.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
  * Class EventStoreSchemaTest
```
Comment (prolic) [inline tests/Schema/SnapshotStoreSchemaTest.php]: Wrong class name

**Comment by prolic** [comment]
Wow, first bc break during beta :-) 
Agree, it must be done.
Please fix the wrong phpdoc class name, then i will merge.

**Comment by codeliner** [comment]
hehe, yeah we are fast with BC breaks (or should we call it agile development ;))

**Comment by codeliner** [comment]
done @prolic

**Pull request merged**